app/viewmodels/journeys/capitalallowances/AssetBasedAllowanceSummary.scala

```diff
@@ -16,33 +16,23 @@
 
 package viewmodels.journeys.capitalallowances
 
-import models.journeys.adjustments.ProfitOrLoss
-import models.journeys.adjustments.ProfitOrLoss.{Loss, Profit}
+import models.journeys.adjustments.NetBusinessProfitOrLossValues
 import play.api.i18n.Messages
 import uk.gov.hmrc.govukfrontend.views.viewmodels.table.Table
 import viewmodels.checkAnswers.{buildTable, buildTableAmountRow, buildTableRow}
 
 object AssetBasedAllowanceSummary {
 
-  // TODO the hardcoded values are used to create the test cases, these can be updated when the values in Car or Asser based Allowance summary will be replaced with API data (SASS-8624)
-
-  def buildCarsAndAssetBasedAllowanceTable(profitOrLoss: ProfitOrLoss)(implicit messages: Messages): Table =
```

Comment:
CarsAndAssetBasedAllowance is only cash, different name for accrual, neither describe the table properly any way so renamed it